content/posts/2022/introducing-launch-on-reana.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Introducing Launch on REANA"
+date: 2022-04-20T14:00:00+02:00
+tags:
+  - API
+  - cluster
+---
+
+We are happy to announce that it is now possible to launch workflows that are
+hosted on external services directly from the REANA web interface, without
+relying on the `reana-client` CLI tool!
+
+<!--more-->
+
+## An overview on how to run workflows
+
+As you might already know, `reana-client` is the command-line tool that can be
+used to interact with the REANA platform. You can use it to create workflows, to
+upload files to their workspaces and to execute them. A tutorial on how to do so
+can be found in the page
+[First example](https://docs.reana.io/getting-started/first-example/) of the
+REANA documentation website.
+
+Another way to manage workflows is to use the Python API. This can be useful if
+you want to interact with REANA from your Python application or script. If you
+are interested in this topic, you should read the blog post on
+[How to run REANA workflows using Python API](/posts/2021/reana-client-python-api/).
+
+However, both of this approaches require installing `reana-client` and setting
+it up. They also require to have all the files needed by the workflow locally,
+so that they can be uploaded to its workspace. All of this can be a hassle in
```

Comment:
About the third issue, `its` is referring to the workflow and not to the files. Should I rewrite the sentence to make it more clear?

Reply:
Probably. What about this for the whole paragraph?

"""
But, both approaches require installing `reana-client` and setting it up. You will also need to upload the necessary files for the workflow. All this can be a hassle in some cases. For example, if the workflow is available online and you only want to run it.
"""